### PR TITLE
pixelfed: 0.11.5 -> 0.11.8

### PR DESCRIPTION
**Pull request by pacien**
- nixos/pixelfed: cleanup package cache at the very start
- pixelfed: add update script
- pixelfed: fix hash format in update script
- pixelfed: 0.11.5 -> 0.11.8

### CODE REVIEW

**Comment by pacien** [comment]
Supersedes #231911.

**Comment by SebRut** [comment]
Result of `nixpkgs-review pr 238307` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>pixelfed</li>
  </ul>
</details>

**Review by RaitoBezarius — APPROVED** [review]
Confirmed that upgrade worked on my side in production.

File: pkgs/servers/web-apps/pixelfed/default.nix [hunk 1]
```diff
@@ -28,17 +28,20 @@ let
   });
 in package.override rec {
   pname = "pixelfed";
-  version = "0.11.5";
+  version = "0.11.8";
 
   # GitHub distribution does not include vendored files
```
Comment (SuperSandro2000) [inline pkgs/servers/web-apps/pixelfed/default.nix:33]:
```suggestion
```
this does not seem to be true anymore

Comment (pacien) [reply pkgs/servers/web-apps/pixelfed/default.nix]: Removed 👍

**Review by wlcx — APPROVED** [review]
Yep, works great on my instance

**Pull request merged**